src/NzbDrone.Core/Indexers/TorrentRssIndexer/TorrentRssIndexerParserSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Indexers.TorrentRssIndexer
+{
+    public class TorrentRssIndexerParserSettings
```

Comment:
That's just an internal setting, nothing external.
It has to store data somewhere :)